src/appshell/qml/Preferences/internal/AutomaticUpdateSection.qml

```diff
@@ -55,7 +55,7 @@ BaseSection {
     StyledTextLabel {
         width: parent.width
 
-        text: qsTrc("appshell/preferences", "Update checking requires network access. In order to protect your privacy, MuseScore does not store any personal information. See our <a href=\"%1\">privacy policy</a> for more info.")
+        text: qsTrc("appshell/preferences", "Update checking requires network access. In order to protect your privacy, MuseScore Studio does not store any personal information. See our <a href=\"%1\">privacy policy</a> for more info.")
```

Comment:
This one is questionable, as it does not refer to the program but to the organisation. @bkunda It would be great to have your input here.